connectors/sources/sharepoint_online.py

```diff
@@ -1775,3 +1788,13 @@ async def close(self):
 
     def advanced_rules_validators(self):
         return [SharepointOnlineAdvancedRulesValidator()]
+
+    def is_supported_format(self, filename):
```

Comment:
nit: that could live in `utils.py` right? `TIKA_SUPPORTED_FILETYPES` also lives in `utils.py`, so would make sense to me to keep things together.

Reply:
Totally agree however,  I'm not sure this is the right moment to move to `utils.py`. The similar code lives in `sharepoint_server.py` and `TIKA_SUPPORTED_FILETYPES ` should not live in `utils.py` either :).